Application/src/tracker/python/test_trex_sam3_interface.py

```
@@ -261,29 +261,40 @@ class FakeOffset:
-        return self._scales
+    def tile_geometries(self):
+        return self._geometries
 
 
 class FakeSam3Input:
     def __init__(self, images, orig_ids, offsets, scales, prompts_per_image):
-        self._base = FakeBaseInput(images, orig_ids, offsets, scales)
+        self._base = FakeBaseInput(
+            images,
+            orig_ids,
+            [FakeTileGeometry(offset, scale) for offset, scale in zip(offsets, scales)])
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)